EIPS/eip-4362.md

```diff
@@ -0,0 +1,270 @@
+
+## Motivation
+
+SIWE CapGroks unlock integration of protocols and/or APIs for developers by reducing user friction, onchain state and increasing security by introducing informed consent and deterministic capability objects on top of Sign-In With Ethereum (EIP-4361).
+
+## Specification
+
+This specification has three different audiences:
+- Web3 application developers that want to integrate CapGroks to authenticate with any protocols and APIs that support object capabilities.
+- Protocol or API developers that want to learn how to define their own CapGroks.
+- Wallet implementers that want to improve the UI for CapGroks.
+
+### Terms and Definitions
+
+- CapGrok - A SIWE Message complying with this specification, i.e., containing at least one CapGrok URI in the `Resources` section and the corresponding human-readable CapGrok Statement appended to the SIWE `statement`.
+- CapGrok URI - A type of URI under a certain namespace that resolves to a CapGrok Details Object.
+- CapGrok Details Object - A JSON object describing the actions and optionally the resources associated with a CapGrok Capability under a certain namespace.
+- Resource Service (RS) - The entity that is providing third-party services for the Ethereum account.
+- SIWE Client (SC) - The entity initiating the SIWE authentication and CapGrok flow.
+- Relying Party (RP) - same as SC in the context of authentication.
+
+### Overview
+
+This specification defines the following:
+- CapGrok SIWE Extension
+- CapGrok Capability
+   - CapGrok URI Scheme
+   - CapGrok Details Object Schema
+- CapGrok Translation Algorithm
+- CapGrok Verification
+
+### CapGrok SIWE Extension
+
+A CapGrok is a EIP-4361 message following a specific format that allows an Ethereum account to delegate a set of CapGrok Capabilities to a delegee through informed consent. Each CapGrok Capability MUST be represented by an entry in the `Resources` array of the SIWE message that MUST deterministically translate the CapGrok Capability in human-readable form to the `statement` field in the SIWE message using the CapGrok Translation Algorithm.
+
+The following SIWE message fields are used to further define (or limit) the scope of all CapGrok Capabilities:
+- The `URI` field MUST specify the delegee as a URI, e.g., `https://example.com`, `did:key:z6MkhaXgBZDvotDkL5257faiztiGiC2QtKLGpbnnEGta2doK`. It is expected that the RS authenticates the delegee before invoking an action for the CapGrok Capability.
```

Comment:
EIP-4361 defines this field as a valid URI. Do we need to restate that requirement here?

Reply:
@SamWilsn I agree. I updated the PR accordingly.

Reply:
@bumblefudge added "intended delegee". Authentication is out-of-scope and is up to the RS.